src/olympia/addons/models.py

```diff
@@ -954,6 +954,12 @@ def current_version(self):
             pass
         return None
 
+    @amo.cached_property(writable=True)
+    def latest_unlisted_version(self):
+        """Shortcut property for Addon.find_latest_version(
+        channel=RELEASE_CHANNEL_UNLISTED)."""
```

Comment:
At the moment it's only used by the serializer. `latest_listed_version` might make sense but I didn't want to make too many modifications outside the API, so I figured we'd revisit that as we make more progress and cleanups in devhub